handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,75 @@
+# <Product> direction
+
+This page outlines the vision, strategy and goals of the <Product> team.
+
+# Vision
+
+#### 1 Year vision
+
+> Some vision
+
+#### 3 Year vision
+
+> Some vision
+
+#### 10 year vision
+
+> Some vision
+
+## Mission
+
+TODO
+
```

Comment:
```suggestion

# Guiding principles

Identify what key principles, insights, and decisions are important to the strategy or a set of key hypotheses and set of key questions to validate or invalidate them.

Describe the target by which we prioritize:
- Market or audience
- Personas 
- Jobs or Use Cases
- Industries
- What matters to us when we prioritize products and features
- Contextual things that matter to Sourcegraph or the product

```

Reply:
I'm a MASSIVE fan of the guiding principles to steer the team.

Reply:
@dan-mckean where does this belong in your opinion? Is it in strategy? Or rather right under mission?

Reply:
Reordered to match your suggestion. Sounds like the Guiding Principles are more relevant in the strategy though, as they are interdependent with the Themes

Reply:
If guiding principles are indeed relevant to both themes and vision, then I agree the right place for them is out of the three year strategy section and next to mission.

Reply:
Another idea @malomarrec, maybe there just doesn't need to be a "3 year strategy and themes" heading?

- Strategy could be combined with Mission into "Mission & Strategy" up at the top
- "Guiding Principles" could be added just below "Mission & Strategy" at the top level
- "Themes" could be moved inside of "What's next & why"

I made a PR https://github.com/sourcegraph/about/pull/4600/files with the changes to consider.

Reply:
Resolving since I merged in that PR.